phpstan.neon

```diff
@@ -287,14 +287,31 @@ parameters:
             paths:
                  - packages/monorepo-builder/packages/split/src/FileSystem/DirectoryToRepositoryProvider.php
 
-
         -
             message: '#Instead of container injection, use specific service#'
             paths:
                  - packages/package-builder/src/Parameter/ParameterProvider.php # 26
                  - packages/symfony-static-dumper/src/HttpFoundation/ControllerContentResolver.php # 37
 
         -
-            message: '#Use namespaceName on Class_ node#'
+            message: '#Property with protected modifier is not allowed\. Use interface instead#'
             paths:
-                 - packages/coding-standard/src/Rules/NoSuffixValueObjectClassRule.php
+                - packages/coding-standard/src/Fixer/ArrayNotation/AbstractArrayFixer.php # 22
+                - packages/coding-standard/src/Fixer/ArrayNotation/AbstractArrayFixer.php # 27
+                - packages/coding-standard/src/Fixer/ArrayNotation/AbstractArrayFixer.php # 32
+                - packages/coding-standard/src/Rules/AbstractPrefferedCallOverFuncRule.php # 18
+                - packages/easy-coding-standard/src/Application/AbstractFileProcessor.php # 16
+                - packages/easy-coding-standard/src/Console/Command/AbstractCheckCommand.php # 25
+                - packages/easy-coding-standard/src/Console/Command/AbstractCheckCommand.php # 30
+                - packages/easy-coding-standard/src/Console/Command/AbstractCheckCommand.php # 35
+                - packages/monorepo-builder/packages/merge/src/ComposerKeyMerger/AbstractComposerKeyMerger.php # 15
+                - packages/monorepo-builder/packages/merge/src/ComposerKeyMerger/AbstractComposerKeyMerger.php # 20
+                - packages/monorepo-builder/packages/release/src/ReleaseWorker/AbstractMutualDependencyReleaseWorker.php # 18
+                - packages/monorepo-builder/packages/release/src/ReleaseWorker/AbstractMutualDependencyReleaseWorker.php # 23
+                - packages/monorepo-builder/packages/release/src/ReleaseWorker/AbstractMutualDependencyReleaseWorker.php # 28
+                - packages/monorepo-builder/packages/release/src/ReleaseWorker/AbstractMutualDependencyReleaseWorker.php # 33
+                - packages/changelog-linker/tests/ChangeTree/ChangeFactory/AbstractChangeFactoryTest.php # 17
+                - packages/changelog-linker/tests/ChangeTree/ChangeFactory/AbstractChangeFactoryTest.php # 26
+                - packages/monorepo-builder/packages/merge/tests/ComposerJsonDecorator/AbstractComposerJsonDecoratorTest.php # 18
+                - packages/package-builder/src/Tests/AbstractKernelTestCase.php # 28
+                - packages/package-builder/src/Tests/AbstractKernelTestCase.php # 33
```

Comment:
@samsonasik Could you follow up with PR, where most these are refactored to either constant or interface method?

Looking at the errors, there should be skipped:

- https://github.com/symplify/symplify/blob/496ae76febc6011fe48ffdd0e0d118531408682c/packages/easy-coding-standard/src/Console/Command/AbstractCheckCommand.php#L25 - if class is an `abstract` and the property is passed as a dependency via constructor

- https://github.com/symplify/symplify/blob/496ae76febc6011fe48ffdd0e0d118531408682c/packages/package-builder/src/Tests/AbstractKernelTestCase.php#L25-L33 - if property is `static` and Container/Kernel type, its most likely a performance cache

Reply:
I will try

Reply:
Thank you